Settings.c

```diff
@@ -307,6 +313,7 @@ Settings* Settings_new(int initialCpuCount) {
    this->updateProcessNames = false;
    this->showProgramPath = true;
    this->highlightThreads = true;
+   this->highlightDelaySecs = DEFAULT_HIGHLIGHT_SECS;
```

Comment:
Shouldn't `highlightChanges` also be initialized (to `false`) when setting the defaults?